packages/core/src/components/date-picker/date-picker.scss

```diff
@@ -9,97 +9,173 @@
 
 @import 'common-variables';
 
-.header {
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-}
-
-.selector {
-  flex-basis: 100%;
-  display: flex;
-  align-items: center;
-  justify-content: center;
-  padding: $tiny-space $default-space;
-
-  .dropdown {
-    color: var(--theme-menu-item--color);
-    font-size: 14px;
-    line-height: 20px;
-  }
-
-  .fontSize {
-    font-size: 16px;
-  }
-
-  .capitalize {
-    text-transform: capitalize;
+:host {
+  .header {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
   }
 
-  .arrowYear {
+  .selector {
+    flex-basis: 100%;
     display: flex;
-    position: relative;
-    padding: $medium-space $x-large-space;
     align-items: center;
-    cursor: pointer;
+    justify-content: center;
+    padding: $tiny-space $default-space;
 
-    &:hover {
-      background-color: var(--theme-select-list-item--background--hover);
+    .dropdown {
+      color: var(--theme-menu-item--color);
+      font-size: 14px;
```

Comment:
Use relative values throughout file (e.g. rem, common variables)

Reply:
Should be fixed with https://github.com/siemens/ix/issues/131